src/Microsoft.ML.HalLearners/ProjectionCatalog.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms.Projections;
+
+namespace Microsoft.ML
+{
+    public static class ProjectionCatalog
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="VectorWhiteningEstimator"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="inputColumn">Name of the input column.</param>
+        /// <param name="outputColumn">Name of the column resulting from the transformation of <paramref name="inputColumn"/>. Null means <paramref name="inputColumn"/> is replaced. </param>
+        /// <param name="kind">Whitening kind (PCA/ZCA).</param>
+        /// <param name="eps">Whitening constant, prevents division by zero.</param>
+        /// <param name="maxRows">Maximum number of rows used to train the transform.</param>
+        /// <param name="pcaNum">In case of PCA whitening, indicates the number of components to retain.</param>
+        public static VectorWhiteningEstimator VectorWhiten(this TransformsCatalog.ProjectionTransforms catalog, string inputColumn, string outputColumn,
```

Comment:
>VectorWhiten [](start = 47, length = 12)

do you feel like writing a sample for it, and referencing it from here?

Reply:
Good point! I'll do that :)